e2e/test/scenarios/admin-2/whitelabel.cy.spec.js

```diff
@@ -275,24 +297,34 @@ describeEE("formatting > whitelabel", () => {
 
       main().findByText("Please make sure this is a valid URL").should("exist");
 
-      getHelpLinkCustomDestinationInput().type("examp");
+      getHelpLinkCustomDestinationInput().type("example");
```

Comment:
It doesn't make any difference here, but I thought we originally missed the full name.